select.scss

```diff
@@ -0,0 +1,127 @@
+// Select
+// ===
+
+
+// Configurable Variables
+// ---
+
+// Whether to draw the default down-arrow icon.
+$select__draw-caret: true !default;
+
+
+// Dependencies
+// ---
+
+@import 'stencil-variables/variables';
+@import 'spline/dist/utilities/triangle';
+@import 'level/level';
+
+
+// Root
+// ---
+
+.c-select {
```

Comment:
I took out the background since that could be argued to not be strictly necessary for the component to function. Maybe going too far? Not sure, but trying to default to fewer styles than to more.